automated install/basic-install.sh

```diff
@@ -988,12 +1122,24 @@ setDNS() {
                 # and continue the loop.
                 DNSSettingsCorrect=False
             else
-                # Otherwise, show the DNS setting to the user, and break the loop if they confirm them.
-                if (whiptail --backtitle "Specify Upstream DNS Provider(s)" --title "Upstream DNS Provider(s)" --yesno "Are these settings correct?\\n    DNS Server 1:   $PIHOLE_DNS_1\\n    DNS Server 2:   ${PIHOLE_DNS_2}" "${r}" "${c}"); then
-                    DNSSettingsCorrect=True
-                else
-                    DNSSettingsCorrect=False
-                fi
+                dialog --no-shadow --keep-tite \
+                    --backtitle "Specify Upstream DNS Provider(s)" \
+                    --title "Upstream DNS Provider(s)" \
+                    --yesno "Are these settings correct?\\n\\tDNS Server 1:\\t${PIHOLE_DNS_1}\\n\\tDNS Server 2:\\t${PIHOLE_DNS_2}" \
+                    "${r}" "${c}" && result=0 || result=$?
+
+                case ${result} in
+                    "${DIALOG_OK}")
+                        DNSSettingsCorrect=True
+                        ;;
+                    "${DIALOG_CANCEL}")
+                        DNSSettingsCorrect=False
+                        ;;
+                    "${DIALOG_ESC}")
+                        printf "  %bEscape pressed, exiting installer at DNS Settings%b\\n" "${COL_LIGHT_RED}" "${COL_NC}"
```

Comment:
```suggestion
                        printf "   %bEscape pressed, exiting installer at DNS Settings%b\\n" "${COL_LIGHT_RED}" "${COL_NC}"
```

Reply:
I think it should be only 2 spaces before the `%b`

```sh
  printf "  %bEscape ..."  
or 
  printf "  %b Escape ..."  
``` 
you added a third space.